### DISCUSSION TITLE
Indicator Window height

### DISCUSSION THREAD

**Discussion by OllieFeraher** [kernc/backtesting.py#704] [Q&A]
Hi,

Is there a way I can increase the height of the indicator window?

Kind regards

**Comment by kernc** [comment]
In your own copy of the software, at this position:
https://github.com/kernc/backtesting.py/blob/65f54f6819cac5f36fd94ebf0377644c62b4ee3d/backtesting/_plotting.py#L271-L272

**Reply by OllieFeraher** [reply]
Worked perfectly.

THANK YOU <3
